addons/sale_project/models/sale_order.py

```diff
@@ -128,12 +128,18 @@ class SaleOrderLine(models.Model):
 
     project_id = fields.Many2one(
         'project.project', 'Generated Project',
+        compute='_compute_project_id', store=True,
```

Comment:
from this:

https://github.com/odoo/odoo/blob/0ae818cca87bd8433a0783797309771125772d86/addons/sale_project/models/sale_order.py#L29-L32

it seems like the user should be able to select a project, so the field should probably at least be `readonly=False`

Reply:
They're able to select it on the SO, but this compute method only affect the order's lines, which is assumed to be generated or provided by the SO rather than user-selected.